### ISSUE TITLE
[ansible/artifactory] default admin credentials are not used/applied

### ISSUE THREAD

**Issue by olwe0002** [jfrog/JFrog-Cloud-Installers#294]
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): both
**Which installer**: ansible
**Which product and version** : ansible &  JFrog Platform 10.12.3 release
**Which operating system and version**(eg: ubuntu & version - 20.4): ubuntu & version - 20.4
**Which product license** (Enterprise/Pro): Enterprise

**What happened**:

The default admin credentials can be defined, default variables are defined for that:

https://github.com/jfrog/JFrog-Cloud-Installers/blob/9d46194903af2cb8303d502a7b199336353ebf66/Ansible/ansible_collections/jfrog/platform/roles/artifactory/defaults/main.yml#L65

but they are not applied on a installation, as this variables are not used at all in any role

**What you expected to happen**:

login with the provided credentials is possible, at least on a new installation

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by itssiri** [comment]
Thanks. We will  take this as an internal enhancement.

**Comment by bbaassssiiee** [comment]
https://github.com/eugene-krivosheyev/ansible-artifactory-role/blob/9bf48844a69f08e6cf8c306c7615750ff5dc18ab/tasks/03-configure-distr.yml#L3

**Comment by olwe0002** [comment]
thx. I tried to use the new collection version, but it failed:
```
TASK [jfrog.platform.artifactory : Set up Artifactory admin account] ***********
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: If you are using a module and expect the file to exist on the remote, see the remote_src option
fatal: [vm-artifactory-1]: FAILED! => changed=false 
  msg: |-
    Could not find or access 'bootstrap.creds'
    Searched in:
            /root/.ansible/collections/ansible_collections/jfrog/platform/roles/artifactory/templates/bootstrap.creds
            /root/.ansible/collections/ansible_collections/jfrog/platform/roles/artifactory/bootstrap.creds
            /root/.ansible/collections/ansible_collections/jfrog/platform/roles/artifactory/tasks/templates/bootstrap.creds
            /root/.ansible/collections/ansible_collections/jfrog/platform/roles/artifactory/tasks/bootstrap.creds
            /builds/alm/artifactory/jfrog-platform-on-openstack/ansible/playbooks/./templates/bootstrap.creds
            /builds/alm/artifactory/jfrog-platform-on-openstack/ansible/playbooks/./bootstrap.creds on the Ansible Controller.
    If you are using a module and expect the file to exist on the remote, see the remote_src option
```

root cause is that the template [file](https://github.com/jfrog/JFrog-Cloud-Installers/blob/master/Ansible/ansible_collections/jfrog/platform/roles/artifactory/templates/bootstrap.creds.j2) is name `bootstrap.creds.j2`, but in the [task](https://github.com/jfrog/JFrog-Cloud-Installers/blob/master/Ansible/ansible_collections/jfrog/platform/roles/artifactory/tasks/install.yml#L173) you are using `bootstrap.creds` as template name

please fix

**Comment by bbaassssiiee** [comment]
Thanks for your report! I'll create a hotfix

**Comment by olwe0002** [comment]
@bbaassssiiee additionally, the `dest` is wrong, as the file name is missing. it doesn't work to use a directory there. Fails with `msg: Destination .../artifactory/var/etc/access not writable`

**Comment by bbaassssiiee** [comment]
This issue is closed, please continue this review in [Pull Request 340](https://github.com/jfrog/JFrog-Cloud-Installers/pull/340)

**Comment by chukka** [comment]
@bbaassssiiee @olwe0002  Thanks for the PR and fixed via #343
